src/Chainweb/Pact/PactService.hs

```diff
@@ -1122,9 +1122,11 @@ getCheckpointer :: PactServiceM cas Checkpointer
 getCheckpointer = view (psCheckpointEnv . cpeCheckpointer)
 
 -- | temporary gas model without sizing
-tableGasModelNoSize :: GasCostConfig -> GasModel
-tableGasModelNoSize gasConfig =
+_tableGasModelNoSize :: GasCostConfig -> GasModel
+_tableGasModelNoSize gasConfig =
```

Comment:
Since we've switched to the actual gas table, this should be removed, no?

Reply:
Yes this should be removed entirely

Reply:
deleted in latest commit